content/en/docs/concepts/workloads/pods/disruptions.md

```diff
@@ -270,6 +270,10 @@ deleted. In such a situation, after some time, the
 Pod disruption condition will be cleared.
 {{< /note >}}
 
+When the `PodDisruptionConditions` feature gate is enabled PodGC, along with
```

Comment:
nit: Mentioning the pod garbage collector and the acronym here would be helpful. 
```suggestion
When the `PodDisruptionConditions` feature gate is enabled, along with
```